docs/userguide/configuration.rst

```diff
@@ -642,6 +642,16 @@ Default: No compression.
 Optional compression method used for task results.
 Supports the same options as the :setting:`task_serializer` setting.
 
+.. setting:: result_extended
+
+``result_extended``
+~~~~~~~~~~~~~~~~~~~~~~
+
+Default: ``True``
```

Comment:
Perhaps this option should use `False` as default, for backwards compatibility.

Reply:
Done. thanks!